openfecwebapp/templates/legal-enforcement-landing.html

```diff
@@ -62,15 +62,13 @@
       If the Commission fails to reach a conciliation agreement with a respondent, the Commission may choose 
       to pursue the matter in federal court.
     </p>
-    <p>
-      Search or browse closed MURs.
-    </p>
     <div class="u-no-print">
       <div class="slab slab--neutral slab--inline">
         <div class="container">
           <div class="row">
             <div class="usa-width-one-whole">
               <form action="{{ url_for('murs') }}" autocomplete="off" class="search__container  js-search">
+                <label class="label">Search or browse MURs</label>
```

Comment:
This `<label>` needs `for="search-input"` so it's associated with the input explicitly.